prusa_slicer_post_processing_script.py

```diff
@@ -604,15 +625,15 @@ def spotFeaturePoints(self,featureName:str,splitAtWipe=False,includeRealStartPt=
             if featureName in ftype:
                 if includeRealStartPt and idf>0:
                     sp=self.getRealFeatureStartPoint(idf)
-                    if sp:pts.append(sp)       
+                    if sp:pts.append(sp)
                 for line in lines:
                     if "G1" in line and (not isWipeMove):
                         if (not "E" in line) and travelstr in line and splitAtTravel:
                             #print(f"Layer {self.layernumber}: try to split feature. No. of pts before:",len(pts))
                             if len(pts)>=2:#make at least 1 ls
                                 parts.append(pts)
-                                pts=[]# update self.features... TODO
-                        elif "E" in line:     #maybe fix error of included travel moves? 
+                            pts=[]# update self.features... TODO
```

Comment:
The other half of the fix for #56. This one removes the included travel moves in the Hilbert curves by always clearing single points when we travel.